cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,54 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
-
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+  it('should display the total number of pages when new rows are added', () => {
+    cy.createWorkers(4, 11);
+    cy.get('.-totalPages').should('contain.text', '2');
+  });
+  it('should allow rows count selection', () => {
+    cy.createWorkers(4, 11);
+    cy.get('select').select('20 rows');
+    cy.get('.rt-td').should('contain.text', 'worker№' + 11);
```

Comment:
```suggestion
    cy.get('.rt-td').should('contain.text', 'worker№' + 11);
```
this assertion doesn't check the amount of rows in the table